app/controllers/reservations_controller.rb

```diff
@@ -382,7 +382,7 @@ def checkout_email
   end
 
   def checkin_email
```

Comment:
These methods nominally get called in the `current_reservations`, `receipt`, and reservation `show` views, but have been commented out for some time. I think it's time we re-enable them :smile:. Note that we'll have to update the routes as well.